package.json

```diff
@@ -7,7 +7,7 @@
     "client": "vite --config vite.config.js",
     "clientinstall": "npm install --prefix client",
     "server": "node index.js",
-    "start": "concurrently 'npm run server' 'npm run client'",
+    "start": "concurrently \"npm run server\" \"npm run client\"",
```

Comment:
Can we remove from this PR and add to a separate pull request?

Reply:
Removed from this PR, will be raising new PR for that.